classes/mender-install.bbclass

```diff
@@ -1,6 +1,10 @@
 PREFERRED_VERSION_go_cross = "1.6%"
 
-#Add support for systemd
+# Add support for systemd
+#
+# Note that a very common error with systemd is that the system hangs during the
+# boot process while looking for devices. This is almost always because the
+# kernel feature CONFIG_FHANDLE is not enabled.
 DISTRO_FEATURES_append = " systemd"
```

Comment:
Right now I'm going for the approach where, if you inherit `mender-full` it produces error if you haven't enabled systemd, but you can omit that by selectively inheriting only the mender classes you need. For QEMU it will not produce error, but will always enable systemd, for convenience. I think this is a good usability vs surprise compromise. Do you agree?